articles/architecture-scenarios/b2c/tenant-architecture.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Tenant Architecture
+description: Learn how to create Auth0 tenants for one or more environments
+toc: true
+topics:
+    - b2c
+    - ciam
+    - tenants
+contentType: concept
+useCase:
+  - tenant-architecture
+---
+# Tenant Architecture
+
+Everything at Auth0 begins with the creation of a tenant. The tenant is where you will configure everything related to your use of Auth0.
+
+You can (and are encouraged to) create more than one tenant. In addition to using different tenants for different environments (for example, you can have a tenant for development and a tenant for production), you can use different tenants to isolate your user domains (determining which user domains you'll need is an important step of the tenant architecture process).
+
```

Comment:
The following is also important because it is re-affirming that you should also be considering what else is going on in your organization too.
```suggestion
Understanding where your application fits within your organization is also important. Auth0 Accounts and Tenants form the basis for the grouping and structuring of Auth0 assets, and it may be that you’ll need to leverage an existing Auth0 deployment in order to integrate with SSO (Single Sign On), centralized User Profile Management, consolidated billing, etc.
```